http/src/main/java/com/redhat/lightblue/client/http/LightblueHttpClient.java

```diff
@@ -55,18 +70,15 @@ public LightblueHttpClient(String configFilePath) {
      * This constructor will use a copy of specified configuration object.
      */
     public LightblueHttpClient(LightblueClientConfiguration configuration) {
-        this(configuration, LightblueResponse.DEFAULT_MAPPER);
+        this(configuration, DEFAULT_MAPPER);
     }
```

Comment:
In #64 I moved this constant to LightblueResponse thinking that we could have a single default mapper defined and use it rather than individual classes defining their own. Maybe a more centralized location would be better? Thoughts?